src/config/materials.js

```diff
@@ -123,6 +123,19 @@ export default {
 
   filters: {
     fields: {
+      material: {
+        field: 'collectionobjects_common:materialGroupList.material',
+        messages: defineMessages({
```

Comment:
You don't need `messages` here, since they are not being changed from the default config, and they'll be inherited from there.